apps/api/src/modules/etl/events/create.ts

```diff
@@ -84,7 +88,7 @@ export const etlEventsCreate = async (
         hostId,
         name,
         description,
-        guestsByUserId: [...inviteesByUserId],
+        guests: [...inviteesByUserId],
```

Comment:
Same note here -- prefer `guestsByUserId` because it makes it really clear how we are representing the guests